demo/api/src/db/fixtures/generators/public-uploads-fixture.service.ts

```diff
@@ -0,0 +1,22 @@
+import { download, PublicUpload, PublicUploadsService } from "@comet/cms-api";
```

Comment:
`download` makes me :sob:

Reply:
Let's continue the discussion here: https://github.com/vivid-planet/comet/pull/1521